rotkehlchen/tests/unit/decoders/test_weth.py

```diff
@@ -465,3 +490,581 @@ def test_wxdai_wrap(database, gnosis_inquirer, gnosis_accounts):
+            counterparty=CPT_GAS,
+        ),
+        EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=1,
+            timestamp=timestamp,
+            location=Location.BASE,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.RETURN_WRAPPED,
+            asset=A_WETH_BASE,
+            balance=Balance(amount=FVal(amount)),
+            location_label='0x44f29ebE386c409376C66ad268F9Ae595c8C3e76',
+            notes=f'Unwrap {amount} WETH',
+            counterparty=CPT_WETH,
+            address=string_to_evm_address('0x4200000000000000000000000000000000000006'),
+        ),
+        EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=2,
+            timestamp=timestamp,
+            location=Location.BASE,
+            event_type=HistoryEventType.RECEIVE,
+            event_subtype=HistoryEventSubType.NONE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(amount)),
+            location_label='0x44f29ebE386c409376C66ad268F9Ae595c8C3e76',
+            notes=f'Receive {amount} ETH',
+            counterparty=CPT_WETH,
+            address=string_to_evm_address('0x4200000000000000000000000000000000000006'),
+        ),
+    ]
+    assert events == expected_events
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('base_accounts', [['0xf396e7dbb20489D47F2daBfDA013163223B892a0']])
+def test_weth_deposit_base(database, base_inquirer):
+    evmhash = deserialize_evm_tx_hash('0x0d418e4a858ca5faf00c36b685561ca0fdac52ebd10364bf2cb6d7b5969e84e5')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=base_inquirer,
+        database=database,
+        tx_hash=evmhash,
+    )
+    timestamp = TimestampMS(1712239899000)
+    amount, gas_fees = '1.2', '0.000000775794575663'
+    expected_events = [
+        EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.BASE,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(gas_fees)),
+            location_label='0xf396e7dbb20489D47F2daBfDA013163223B892a0',
+            notes=f'Burned {gas_fees} ETH for gas',
+            counterparty=CPT_GAS,
+        ),
+        EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=1,
+            timestamp=timestamp,
+            location=Location.BASE,
+            event_type=HistoryEventType.DEPOSIT,
+            event_subtype=HistoryEventSubType.DEPOSIT_ASSET,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(amount)),
+            location_label='0xf396e7dbb20489D47F2daBfDA013163223B892a0',
+            notes=f'Wrap {amount} ETH in WETH',
+            counterparty=CPT_WETH,
+            address=string_to_evm_address('0x4200000000000000000000000000000000000006'),
+        ),
+        EvmEvent(
```

Comment:
As I said everywhere in this file

```suggestion
        ), EvmEvent(
```